graql/reasoner/roleplayer-attachment.feature

```diff
@@ -59,6 +59,8 @@ Feature: Roleplayer Attachment Resolution
       """
```

Comment:
If we're settled on banning roleplayer attachment in Reasoner, we should delete this entire file. If we change our mind later, we can restore it from version control.

Reply:
File deleted